debug/index.html

```diff
@@ -58,7 +58,8 @@
     zoom: 12.5,
     center: [-77.01866, 38.888],
     style: 'mapbox://styles/mapbox/streets-v8',
-    hash: true
+    hash: true,
+    maxWorkers: 3
```

Comment:
@lucaswoj @mourner I considered "num", but chose "max" -- this was the argument in my head:

`maxWorkers` is really what the user (me) wants to control, whereas `numWorkers` is more of an internal property.  The fact that they are the same right now is an implementation detail; I could imagine the implementation changing to, say, construct/destroy workers on demand (especially given the [possibility of worker pooling](https://github.com/mapbox/mapbox-gl-js/issues/899)).  

That said, I don't feel especially strongly about this, so just lemme know what yall think